app/controllers/admin/api/backend_apis/base_controller.rb

```diff
@@ -17,6 +17,7 @@ def backend_api
   end
 
   def authorize
-    authorize! :manage, BackendApi
+    return unless current_user # provider_key access
+    authorize! :edit, BackendApi
```

Comment:
Why not `authorize! action_name.to_sym, BackendApi` as you did for the other controller? 🤔

Reply:
This is for metrics and mapping rules, not the api backend that owns those. Whoever can _edit_ an api backend (but not  necessarily _create_ and _delete_ those) can also do everything related to the metrics and mapping rules of the backend. Think of it as handling the composing parts of a backend is equivalent to editing the backend ;)